doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -101,11 +101,11 @@ apt-get update \
     ...
 ```
 
-2. The previous step installed g++-7 and gcc-7 packages, which might be different
+2. The previous step installed g++-11.2 and gcc-11.2 packages, which might be different
```

Comment:
This is not true. ubuntu:16.04 is no longer supported by `mkdocker.sh`, ubuntu:18.04 would yield gcc-8 and ubuntu:20.04 would yield gcc-10. gcc-11 is only included for centos.

I have changes to `mkdocker.sh` that would support ubuntu:22.04 with gcc-11, but I don't have access to POWER 9 hardware to test it fully (POWER 8 hardware is not supported by 22.04).

Reply:
Does this help?
https://hyc-runtimes-jenkins.swg-devops.com/label/hw.arch.ppc64le.p9/

Reply:
I'll give that a try.

Reply:
I expect `/usr/local/gcc11/bin/gcc-11.2` only works on centos, what is the value to use for Ubuntu?

Reply:
On Ubuntu:22.04 it's 11.4 (currently).

Reply:
Hopefully the update is good now.